ui/src/components/expandable_container/ExpandableContainer.js

```diff
@@ -16,25 +27,38 @@ export const ExpandableContainer = ({ maxHeight = 300, children }) => {
       className={classNames("expandableContainer", {
         "expandableContainer-isOpen": isExpanded
       })}>
-      <div
-        className="expandableContainer__childWrapper"
-        style={{
-          height: isExpanded
-            ? contentHeight
-            : Math.min(contentHeight, maxHeight)
-        }}>
-        <div ref={contentRef}>{children}</div>
-      </div>
+      <EuiFlexGroup direction="row" gutterSize="xs" alignItems="left">
```

Comment:
Do you still need to wrap this into a flex group?

Reply:
Removed.